apps/mull-ui/src/app/pages/create-event/create-event.tsx

```diff
@@ -146,7 +150,7 @@ const CreateEventPage = ({ history }: CreateEventProps) => {
           },
         });
         updateToast('Event Created', toast.TYPE.SUCCESS);
-        history.push(ROUTES.HOME);
+        history.push(ROUTES.EVENT_BY_ID.replace(':id', `${eventId}`));
```

Comment:
⚠ Careful, the acceptance criteria is to redirect back to `/home/myevents`. However, this currently probably won't work. After you route to `/home/myevents` it seems it always redirects back to `/home/discover`. To fix this, you need to remove the following code from lines 104-106:
```tsx
                <PrivateRoute exact path={ROUTES.HOME}>
                  <Redirect to={ROUTES.DISCOVER.url} />
                </PrivateRoute>
```

Reply:
oh this one is when you created a event, it redirects you to the event itself. Sorry  I misunderstood the specification, it says to redirect to my event, didn't know it's the my events page. I don't think the lines in create-event you highlighted has anything to do with the bug you pointed out

Reply:
It's fixed now :3